bundles/CoreBundle/Resources/translations/cs.extended.json

```diff
@@ -687,7 +687,7 @@
     "custom_login_background_image": "Custom Login Background Image",
     "show_charcount": "Show Character Count",
     "max_length": "Max Length",
-    "reverse_many_to_many_object_relation": "Reverse Many-To-Many Object Relation",
+    "reverse_object_relation": "Reverse Object Relation",
```

Comment:
please revert all translation files except en.extended.json. Translations are managed in POEditor: https://poeditor.com/join/project/VWmZyvFVMH

Reply:
Reverted the files.